### PR TITLE
WT-7730 Shifting the oldest and stable timestamps to match the commit timestamp format.

### PR DESCRIPTION
**Pull request by luke-pearson**

### CODE REVIEW

File: test/cppsuite/test_harness/timestamp_manager.h [hunk 1]
```diff
@@ -50,11 +52,17 @@ class timestamp_manager : public component {
     load() override final
     {
         component::load();
-
-        _oldest_lag = _config->get_int(OLDEST_LAG);
+        int64_t oldest_lag = _config->get_int(OLDEST_LAG);
         testutil_assert(_oldest_lag >= 0);
```
Comment (etienneptl) [inline test/cppsuite/test_harness/timestamp_manager.h:56]: assert on oldest_lag instead ?

Comment (etienneptl) [reply test/cppsuite/test_harness/timestamp_manager.h:56]: We could directly do `_oldest_lag = _config->get_int(OLDEST_LAG);`

File: test/cppsuite/test_harness/timestamp_manager.h [hunk 2]
```diff
@@ -50,11 +52,17 @@ class timestamp_manager : public component {
...
-        _stable_lag = _config->get_int(STABLE_LAG);
+        /* Cast and then shift left to match the seconds position. */
+        _oldest_lag = oldest_lag;
+        _oldest_lag <<= 32;
+
+        int64_t stable_lag = _config->get_int(STABLE_LAG);
         testutil_assert(_stable_lag >= 0);
```
Comment (etienneptl) [inline test/cppsuite/test_harness/timestamp_manager.h:62]: assert on stable_lag instead ?

Comment (etienneptl) [reply test/cppsuite/test_harness/timestamp_manager.h:62]: Why dont we use _stable_lag directly here too ?

File: test/cppsuite/test_harness/timestamp_manager.h (hunk 1 above)
Comment (luke-pearson) [reply test/cppsuite/test_harness/timestamp_manager.h:56]: Those asserts are definitely on the wrong thing, I wanted it to be explicit that get_int is returning an int64_t and our oldest and stable lag values should be uint64_ts.

**Review by etienneptl — APPROVED** [review]
LGTM

**Review by tetsuo-cpp — APPROVED** [review]
LGTM.

**Pull request merged**